README.md

```diff
@@ -22,29 +18,32 @@
 - [megalinter](https://megalinter.io) - Integrated into MegaLinter.
 - [putout](https://github.com/coderaiser/putout) - Integrated into Putout.
+- [Qlty plugin](https://github.com/qltysh/qlty/tree/main/qlty-plugins/plugins/linters/stylelint) - Qlty integration.
 - [reviewdog/action-stylelint](https://github.com/reviewdog/action-stylelint) - Reviewdog GitHub Action.
 
 ## Configs
 
-- [@archoleat/stylelint-config-extended-scss](https://www.npmjs.com/package/@archoleat/stylelint-config-extended-scss) - Archoleat config for SCSS with property sorting.
-- [@primer/stylelint-config](https://www.npmjs.com/package/@primer/stylelint-config) - The GitHub shareable config.
-- [@stylistic/stylelint-config](https://www.npmjs.com/package/@stylistic/stylelint-config) - The stylistic shareable config for `@stylistic/stylelint-plugin`.
-- [@wordpress/stylelint-config](https://www.npmjs.com/package/@wordpress/stylelint-config) - WordPress CSS Coding Standards shareable config.
-- [stylelint-config-clean-order](https://www.npmjs.com/package/stylelint-config-clean-order) - Order styles with stylelint-order.
-- [stylelint-config-hudochenkov](https://www.npmjs.com/package/stylelint-config-hudochenkov) - Personal configs use stylelint-order.
-- [stylelint-config-idiomatic-order](https://www.npmjs.com/package/stylelint-config-idiomatic-order) - Order styles based on idiomatic-css.
-- [stylelint-config-prettier-scss](https://www.npmjs.com/package/stylelint-config-prettier-scss) - Turns off conflicting rules when using Prettier for SCSS.
-- [stylelint-config-property-sort-order-smacss](https://www.npmjs.com/package/stylelint-config-property-sort-order-smacss) - Property Sort Ordering based on the SMACSS methodology.
-- [stylelint-config-recess-order](https://www.npmjs.com/package/stylelint-config-recess-order) - Sorts CSS properties the way Recess did and Bootstrap does.
-- [stylelint-config-recommended-vue](https://www.npmjs.com/package/stylelint-config-recommended-vue) - The recommended shareable Vue config for Stylelint.
+### Custom syntaxes
+
 - [stylelint-config-sass-guidelines](https://www.npmjs.com/package/stylelint-config-sass-guidelines) - Sass guidelines config.
 - [stylelint-config-standard-scss](https://www.npmjs.com/package/stylelint-config-standard-scss) - Standard config for SCSS.
-- [stylelint-config-standard](https://www.npmjs.com/package/stylelint-config-standard) - Standard config for CSS.
-- [stylelint-config-two-dash-bem](https://www.npmjs.com/package/stylelint-config-two-dash-bem) - BEM Convention zero dependency rule enforcing Two Dashes style for Stylelint.
-- [stylelint-config-wikimedia](https://www.npmjs.com/package/stylelint-config-wikimedia) - Wikimedia CSS Coding Standards shareable config.
+- [stylelint-config-recommended-vue](https://www.npmjs.com/package/stylelint-config-recommended-vue) - The recommended shareable Vue config for Stylelint.
```

Comment:
[suggestion] Does moving `stylelint-config-recommended-vue` before `stylelint-config-sass-guidelines` make sense for alphabetical order?